Keas.Mvc/Controllers/SuperController.cs

```diff
@@ -5,6 +5,7 @@
 namespace Keas.Mvc.Controllers
 {
     [Authorize]
+    [AutoValidateAntiforgeryToken]
```

Comment:
@jcubbage please check that this doesn't break all of our APIs.  I don't think we are passing tokens along w/ API calls and if not this would break everything.  Give the asset pages a once over before you merge

Reply:
@srkirkland Just use [IgnoreAntiforgeryTokenAttribute] where needed?

Reply:
<in Kirk screaming Kahn voice>JJJJJJJAAAAAASSSSSOOOONNNN!!!!!</voice>

Reply:
Totally broke all the React API calls.

Reply:
@jcubbage 
An example if it helps later on:

```const antiforgery = $("input[name='__RequestVerificationToken']").val();
    $.post({
      url: postUrl,
      data: { model: order, __RequestVerificationToken: antiforgery }
    })```